modules/pin/static/js/togglesidebar.js

```diff
@@ -0,0 +1,3 @@
+function toggleSidebar(ref){
```

Comment:
Why is this function needed here? This is supposed to be used for the navigation bar, which already existed in the base js folder.